xml/System.Formats.Cbor/CborReader.xml

```diff
@@ -237,9 +292,27 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Reads the next data item as a byte string (major type 2).</summary>
```

Comment:
Why is `byte[]` called byte string here?

Reply:
The method is called `ReadByteString`, so I assume the byte array contains a string represented as bytes.